example/account_metadata.tf

```diff
@@ -0,0 +1,26 @@
+module "account_metadata" {
```

Comment:
example module call that can be copy pasted to your module

Reply:
@jstehlik-ohpen  I would create a section in the README.md with something like template and link to the files in the example/* folder so when developer are reading the readme can go there directly

Reply:
Great idea!
I will add it before merging.

Reply:
@cangulomascarell Added link in How-to section